packages/test/test-end-to-end-tests/src/test/handleValidation.spec.ts

```diff
@@ -69,10 +76,59 @@ function treeSetup(dds: ITree) {
 	return view;
 }
 
-async function setup(
-	getTestObjectProvider: (options?: ITestObjectProviderOptions) => ITestObjectProvider,
-	apis: CompatApis,
-) {
+interface aDDSType {
```

Comment:
i was considering renaming these to `HandleStorage` and `HandleStorageFactory`, as I think it'd make it a bit clearer. If reviewer is ok with it, I can do a bunch of renames in a separate commit to make this whole file more consistent in naming (didn't want to make review too challenging, it's already a bit annoying as I moved some code around)

Reply:
maybe just do it in a follow up to keep this change reasonable. should be quick to review and approve a simple rename.